### PR TITLE
Implements #74

### PR DESCRIPTION
**Pull request by WCBeckley**
Very basics:
When you make a model change call
flask db migrate
which creates essentially a commit in the migrations folder. Check that the migration does what you want it to do (add this row, remove that row etc.)

When you want to actually change the database call
flask db upgrade
which will modify the actual database.

### CODE REVIEW

File: code/transit-be/app/__init__.py [hunk 1]
```diff
@@ -1,12 +1,13 @@
...
 from flask_sqlalchemy import SQLAlchemy
 from flask_marshmallow import Marshmallow
+from flask_migrate import Migrate
 
 import logging
 
 db = SQLAlchemy()
 ma = Marshmallow()
-logging.basicConfig(filename='logging.log', level='debug')
+# logging.basicConfig(filename='logging.log', level='debug')
```
Comment (WCBeckley) [inline code/transit-be/app/__init__.py:10]: This was causing failures for me so I just commented it out. I don't think we were doing anything with it regardless.

File: code/transit-be/migrations/alembic.ini [hunk 2]
```diff
@@ -0,0 +1,45 @@
+# A generic, single database configuration.
```
Comment (WCBeckley) [inline code/transit-be/migrations/alembic.ini:1]: Auto generated

File: code/transit-be/migrations/env.py [hunk 3]
```diff
@@ -0,0 +1,96 @@
+from __future__ import with_statement
```
Comment (WCBeckley) [inline code/transit-be/migrations/env.py:1]: auto generated

File: code/transit-be/migrations/script.py.mako [hunk 4]
```diff
@@ -0,0 +1,24 @@
+"""${message}
```
Comment (WCBeckley) [inline code/transit-be/migrations/script.py.mako:1]: auto generated

File: code/transit-be/app/__init__.py (hunk 1 above)
Comment (phillipmonroe) [reply code/transit-be/app/__init__.py:10]: This error is caused by debug being lowercase instead of upper

**Pull request merged**